omero/sysadmins/whatsnew.txt

```diff
@@ -11,3 +11,6 @@ What's new for OMERO 5.3 for sysadmins
 
 - Windows support discontinued, see :doc:`/sysadmins/windows-migration`.
 
+- OMERO.web has been decoupled from the server and can now be deployed
+  separately. Support for deployment using Apache has been dropped.
+
```

Comment:
Maybe  a note similar to the one http://www.openmicroscopy.org/site/support/omero5.3-staging/sysadmins/unix/install-web.html
under deployment so we people know that there is a solution